apis/infix/atrium-api-infix/src/commonMain/kotlin/ch/tutteli/atrium/api/infix/en_GB/iteratorExpectations.kt

```diff
@@ -28,3 +26,29 @@ infix fun <T : Iterator<*>> Expect<T>.toHave(@Suppress("UNUSED_PARAMETER") next:
  */
 infix fun <T : Iterator<*>> Expect<T>.notToHave(@Suppress("UNUSED_PARAMETER") next: next): Expect<T> =
     _logicAppend { hasNotNext() }
+
+/**
+ * Expects that `this` expectation (an [Iterator]) has a next element and
+ * returns an [Expect] for the next element.
+ *
+ * @return The newly created [Expect] for the next element.
+ *
+ * @sample TODO
+ *
+ * @since 1.3.0
+ */
+infix fun <E, T : Iterator<E>> Expect<T>.next(@Suppress("UNUSED_PARAMETER") next: next): Expect<E> =
```

Comment:
move to iteratorFeatureExtractors.kt (the file does not exist yet, create it please).

Please use `o` as filler object (see other infix function using `o`)

Reply:
replaced `next` with `o` (file is changed though)